### PR TITLE
Combine view command with find command

### PR DESCRIPTION
**Pull request by Gabau**
Done:

 * Combined view and find into command (Takes in field input as well but retains similar functionality to find.) Field input is specific)
 * Somehow #173 got fixed, I haven't identified the cause yet

UG: [link](https://gabau.github.io/tp/UserGuide.html)

### CODE REVIEW

**Review by mweeruien — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/FindCommand.java [hunk 1]
```diff
@@ -117,59 +162,24 @@ public CommandResult executeIndexSearch(Model model) {
...
-     * @param otherFind The other FindCommand object to be checked
-     * @return Whether the otherFind is equal to this
-     */
-    public boolean findByNameIsEquals(FindCommand otherFind) {
-        return (otherFind.getIndex() == INVALID_INDEX && this.index == INVALID_INDEX)
-                && (otherFind.getNamePredicate() != null && this.namePredicate != null)
-                && (this.namePredicate.equals(otherFind.getNamePredicate()));
+                && ((FindCommand) other).namePredicate.equals(this.namePredicate)
+                && ((FindCommand) other).index == index
+                && predicate.equals(((FindCommand) other).predicate));
```
Comment (IrvingHe000) [inline src/main/java/seedu/address/logic/commands/FindCommand.java:182]: Maybe can consider `FindCommand otherCommand = (FindCommand) other;` first?

**Review by IrvingHe000 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/FindCommand.java (hunk 1 above)
Comment (Gabau) [reply src/main/java/seedu/address/logic/commands/FindCommand.java:182]: I do not think it is necessary for 3 field accesses.

**Pull request merged**